src/assets/i18n/en.json

```diff
@@ -116,7 +116,7 @@
     },
     "add": "Add",
     "type": "Type:",
-    "features_added_soon": "Features to be added to this package soon"
+    "features_added_soon": "Added later to this package"
```

Comment:
@zainzafar90 please update this in all the languages, otherwise it would be difficult to track later.

Reply:
okay